### PR TITLE
feat: add sponsor section

### PR DESCRIPTION
**Pull request by ratnakar5938**
### Your checklist for this pull request
🚨Please review the [guidelines for contributing](../CONTRIBUTING.md) to this repository.

- [x] Make sure you are requesting to **pull a topic/feature/bugfix branch** (right side). Don't request your default branch!
- [x] Make sure you are making a pull request against the **default branch** (left side). Also you should start *your branch* off *default branch*.
- [x] Check the commit's or even all commits' message styles matches our requested structure.
- [x] Check your code additions will fail neither code linting checks nor unit test.
- [x] I have added necessary documentation (if appropriate)

### Description
Please describe your pull request.

❤️Thank you!

### Post merge checklist
- [ ] Follow steps from the [guidelines for contributing](../CONTRIBUTING.md) to this repository.
- [ ] If you are a new contributor, ping in the thread and one of the maintainers will add you to all-contributors list.

### CODE REVIEW

File: src/components/SponsorsSection/Sponsor.jsx [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+        </SponsorContainer>
+
+        <HeadingSection>
+          <Heading2>Gamma Sponsors</Heading2>
+        </HeadingSection>
+
+        <SponsorContainer>
+          <SponsorIcon pic='sponsor' />
+          <SponsorIcon pic='sponsor' />
+          <SponsorIcon pic='sponsor' />
```
Comment (abhibhaw) [inline src/components/SponsorsSection/Sponsor.jsx:57]: Replace these repetitive components with iterating over a list of sponsor pictures

**Review by abhibhaw — CHANGES_REQUESTED** [review]
@CIPHERTron Reloading the page on a slow connection is distorting the design paint. Try to find a fix

**Comment by CIPHERTron** [comment]
> @CIPHERTron Reloading the page on a slow connection is distorting the design paint. Try to find a fix

@abhibhaw I think this happens because Nextjs fetches the styles after rendering the page

https://dev.to/rsanchezp/next-js-and-styled-components-style-loading-issue-3i68
I'm not sure but this might be the solution to our problem. Let me look into it!

**Comment by abhibhaw** [comment]
This should work or maybe we need a preloader?

> > @CIPHERTron Reloading the page on a slow connection is distorting the design paint. Try to find a fix
> 
> @abhibhaw I think this happens because Nextjs fetches the styles after rendering the page
> 
> https://dev.to/rsanchezp/next-js-and-styled-components-style-loading-issue-3i68
> I'm not sure but this might be the solution to our problem. Let me look into it!

**Comment by CIPHERTron** [comment]
> This should work or maybe we need a preloader?
> 
> > > @CIPHERTron Reloading the page on a slow connection is distorting the design paint. Try to find a fix
> > 
> > 
> > @abhibhaw I think this happens because Nextjs fetches the styles after rendering the page
> > https://dev.to/rsanchezp/next-js-and-styled-components-style-loading-issue-3i68
> > I'm not sure but this might be the solution to our problem. Let me look into it!

We are going to need a preloader :')

**Comment by riteshsp2000** [comment]
> > This should work or maybe we need a preloader?
> > > > @CIPHERTron Reloading the page on a slow connection is distorting the design paint. Try to find a fix
> > > 
> > > 
> > > @abhibhaw I think this happens because Nextjs fetches the styles after rendering the page
> > > https://dev.to/rsanchezp/next-js-and-styled-components-style-loading-issue-3i68
> > > I'm not sure but this might be the solution to our problem. Let me look into it!
> 
> We are going to need a preloader :')

Technically speaking a preloader is nothing but again css. You will be using css preloader waiting for css to load. Also I'm using a similar setup of gatsby and styled components on my website, there isn't this issue, might be something with tailwind. Need to check that again, otherwise a preloader might not help

**Comment by CIPHERTron** [comment]
> > > This should work or maybe we need a preloader?
> > > > > @CIPHERTron Reloading the page on a slow connection is distorting the design paint. Try to find a fix
> > > > 
> > > > 
> > > > @abhibhaw I think this happens because Nextjs fetches the styles after rendering the page
> > > > https://dev.to/rsanchezp/next-js-and-styled-components-style-loading-issue-3i68
> > > > I'm not sure but this might be the solution to our problem. Let me look into it!
> > 
> > We are going to need a preloader :')
> 
> Technically speaking a preloader is nothing but again css. You will be using css preloader waiting for css to load. Also I'm using a similar setup of gatsby and styled components on my website, there isn't this issue, might be something with tailwind. Need to check that again, otherwise a preloader might not help
> 

🥲🥲🥲

**Comment by CIPHERTron** [comment]
Heyy @ratnakar5938 , is the mapping done?

**Comment by ratnakar5938** [comment]
> Heyy @ratnakar5938 , is the mapping done?

Yeah it is done

**Comment by ratnakar5938** [comment]
Should I commit it??

**Comment by CIPHERTron** [comment]
> Should I commit it??

yeah sure!

**Comment by CIPHERTron** [comment]
Also, @ratnakar5938, the grid view is not applied. Kindly check that once.
Logos are overlapping in `md`, `sm` & `xs` views. 
Fix that and commit the changes again!

**Comment by ratnakar5938** [comment]
> Also, @ratnakar5938, the grid view is not applied. Kindly check that once.
> Logos are overlapping in `md`, `sm` & `xs` views.
> Fix that and commit the changes again!

Please check now bhaiya i have fixed that issue

**Comment by riteshsp2000** [comment]
@ratnakar5938 Looks all good, one small change, instead of 3 in a row in mobile view, can you make it 2 in a row for mobile view?

**Comment by CIPHERTron** [comment]
Good work @ratnakar5938 
One last thing. Pull upstream main to this i.e `ratnakar-2` branch of yours locally, resolve the conflicts and commit the changes.

**Review by riteshsp2000 — APPROVED** [review]

**Review by abhibhaw — APPROVED** [review]
Good job @ratnakar5938

**Review by CIPHERTron — APPROVED** [review]

**Comment by abhibhaw** [comment]
@riteshsp2000 squash krna tha 😆

**Comment by riteshsp2000** [comment]
Naa bhai squash kyun? Bichare ka pura commits chala jata 🥲

**Comment by CIPHERTron** [comment]
> Naa bhai squash kyun? Bichare ka pura commits chala jata 🥲

So kind XD

**Comment by CIPHERTron** [comment]
@all-contributors add @ratnakar5938 for Code

**Comment by abhibhaw** [comment]
> Naa bhai squash kyun? Bichare ka pura commits chala jata 🥲

ahh the squash pain 🗡️

**Pull request merged**